Packs/GuardiCore/Integrations/GuardiCoreV2/GuardiCoreV2.yml

```diff
@@ -0,0 +1,1284 @@
+category: Deception
+commonfields:
+  id: GuardiCore v2
+  version: -1
+configuration:
+  - name: base_url
+    display: API Server URL
+    required: true
+    defaultvalue: https://example.com/
+    type: 0
+    additionalinfo: null
+  - name: username
+    display: Username for API
+    required: true
+    defaultvalue: null
+    type: 0
+    additionalinfo: null
+  - name: password
+    display: Password for API
+    required: true
+    defaultvalue: null
+    type: 4
```

Comment:
please use the credentials type parameter. https://xsoar.pan.dev/docs/integrations/code-conventions#credentials